<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyTimeTracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>660</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>660</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <widget class="QFrame" name="frame_3">
         <property name="geometry">
          <rect>
           <x>550</x>
           <y>10</y>
           <width>101</width>
           <height>80</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="lbl_idle">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>66</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl_idle_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>66</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>idle secs</string>
          </property>
         </widget>
        </widget>
        <widget class="QFrame" name="frame_4">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>10</y>
           <width>101</width>
           <height>80</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="lbl_active">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>66</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <widget class="QLabel" name="lbl_caption2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>66</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>active</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="timegraph" name="frm_timegraph">
        <property name="minimumSize">
         <size>
          <width>660</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>660</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>timegraph</class>
   <extends>QFrame</extends>
   <header>timegraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
